lib/rules/declaration-property-value-no-unknown/README.md

```diff
@@ -139,3 +139,138 @@ a { top: --foo(0); }
 ```css
 a { top: --foo(10px); }
 ```
+
+### `atRulesSyntax: { "at-rule": { "prelude": "syntax", "descriptors": { "descriptor": "syntax" } } }`
+
+Extend or alter the at-rules syntax dictionary. This option allows you to define custom at-rules or modify existing ones by specifying their expected syntax. Both `prelude` and `descriptors` are optional. The `prelude` is the part of the at-rule that comes after the at-rule name and before the block (if any). The `descriptors` are like properties specific to the at-rule, and are defined similarly to CSS properties.
+
+[CSS Value Definition Syntax](https://github.com/csstree/csstree/blob/master/docs/definition-syntax.md) is used to define the syntax of at-rules, their preludes, and descriptors. If a definition starts with `|` it is added to the existing definition value if any. See the [CSS syntax reference](https://csstree.github.io/docs/syntax/) for default definitions.
+
+Given:
+
+```json
+{
+  "atRulesSyntax": {
+    "example": {
+      "prelude": "<custom-ident>",
+      "descriptors": {
+        "foo": "<number>",
+        "bar": "<color>"
+      }
+    }
+  }
+}
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@example my-ident { foo: 42; bar: red; }
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+@example 10px { foo: 42; bar: red; }
+```
+
+<!-- prettier-ignore -->
+```css
+@example my-ident { foo: red; bar: 42; }
+```
+
+### `cssWideKeywords: [ "keyword", ... ]`
+
+Extend the list of CSS-wide keywords. By default, the rule considers standard CSS-wide keywords like `inherit`, `initial`, `unset`, etc. This option allows you to add custom global keywords that should be accepted as valid property values.
+
+Given:
+
+```json
+{ "cssWideKeywords": ["my-global-value"] }
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { width: my-global-value; }
+```
+
+<!-- prettier-ignore -->
+```css
+a { color: my-global-value; }
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { width: unknown-value; }
+```
+
+<!-- prettier-ignore -->
+```css
+a { color: invalid-keyword; }
+```
+
+## Examples
```

Comment:
From this point on, it seems like duplication.

Reply:
Oh, I was inattentive. Fixed it.
[remove duplicate content](https://github.com/stylelint/stylelint/pull/8124/commits/a7581669c00d38e435f4ef3212f84463aab7458c)